ui/src/pages/ViewEvent.jsx

```diff
@@ -8,10 +8,20 @@ import Image from 'components/image/Image';
 import Title from 'components/text/Title';
 import Header from 'components/header';
 import Section from 'components/section';
+import AddToTrip from './AddToTrip';
 
 import requireAuth from './requireAuth';
 
 class ViewEvent extends React.Component {
+
+  constructor(props)
+  {
+    super(props);
+    this.state = {showAddPage : false};
+
+    this.showAddPage = this.showAddPage.bind(this);
```

Comment:
Remove this and bind the function inline (follows how we have done it in the other components)